c-eyes@anaximeno/files/c-eyes@anaximeno/6.2/applet.js

```diff
@@ -41,6 +41,11 @@ function _(text) {
 }
 
 
+// Mark this for translation since it won't be
+// marked by the translation tool
+const _DEFAULT_TOOLTIP = _("Hey, I saw that!");
```

Comment:
Maybe we can configure better string detection in the settings file. This is a fine workaround for now though.

Reply:
Yes, I was thinking about it too, because if you set the line for translation on the pot file it will disappear when running the makepot script. I was wondering if it was possible to add a marker above the text on the pot file signaling for it not to be removed even if the text is not found on the codes.

Reply:
We don't need to fix it right away but long-term, I think we can fix the detection rather than encourage manual string placements.